org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java

```diff
@@ -129,6 +129,8 @@
     private final int POLLING_PERIOD_DEFAULT = 1800;
     private int pollingPeriod = POLLING_PERIOD_DEFAULT;
 
+    private final int POLLING_OR_REPORTING_FACTOR = 2;
```

Comment:
I think this should be >2 to avoid mistriggers. you want to allow one loss, and setting this to 2 doesn't provide that - or at least it provides absolutely no margin.

Reply:
Right, I was just afraid that 3 might be too large as the default is 7200 seconds = 2h maximum reporting time, times 3 is 6 hours until it goes offline.

But if we only want to allow **one** loss, we do not need 3, we could also use a slightly larger number than 2 and round it to an INT, WDYT?

Reply:
Yes, I didn't expect this to be 3 - I thought 2.1, or something like that. Or you could use 2, and then add 30 seconds or something similar - just something to give some margin.